float32 threshold 

---
bool result
